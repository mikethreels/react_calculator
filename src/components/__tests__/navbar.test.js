/* eslint-disable react/forbid-foreign-prop-types */
import React from 'react';
import renderer from 'react-test-renderer';
import { BrowserRouter } from 'react-router-dom';
import NavBar from '../navbar';

describe('Test suits for MyComponentWithLink', () => {
  it('should match with snapshot', () => {
    const tree = renderer
      .create(
        <BrowserRouter>
          <NavBar />
        </BrowserRouter>,
      )
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});
