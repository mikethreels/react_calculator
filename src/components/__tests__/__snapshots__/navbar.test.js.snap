// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test suits for MyComponentWithLink should match with snapshot 1`] = `
<div>
  <h1>
    Math Magicians
  </h1>
  <ul>
    <li>
      <a
        href="/"
        onClick={[Function]}
      >
        Home
      </a>
      <a
        href="/calculator"
        onClick={[Function]}
      >
        Calculator
      </a>
      <a
        href="/quote"
        onClick={[Function]}
      >
        Quote
      </a>
    </li>
  </ul>
</div>
`;
